tests/integration/utils.py

```diff
@@ -44,3 +56,71 @@ def run_cmd_and_pipe_stderr(command: str) -> List[str]:
 
     print(*stderr, sep="\n")
     return stderr
+
+
+def _get_test_params() -> List[CoreParameter]:
+    param = CoreParameter()
+    ts_param = AreaMeanTimeSeriesParameter()
+
+    m2d_param = MeridionalMean2dParameter()
+    m2d_param.plevs = [
+        200.0,
+        500.0,
+    ]
+    z2d_param = ZonalMean2dParameter()
+    z2d_param.plevs = [
+        200.0,
+        300.0,
+    ]
+
+    enso_param = EnsoDiagsParameter()
+    enso_param.test_name = "e3sm_v1"
+
+    params = [param, ts_param, m2d_param, z2d_param, enso_param]
+
+    return params
+
+
+def _convert_cfg_to_param_objs(cfg_path: str) -> List[CoreParameter]:
+    """Convert diagnostic cfg entries to parameter objects.
+
+    NOTE: ast.literal_eval is not considered "safe" on untrusted data.
+    The reason why it is used is because `configparser.ConfigParser`
+    doesn't work well with parsing Python types from strings in
+    `.cfg` files, resulting in things such as nested strings or string
+    representation of lists. Since we are only calling literal_eval on
+    `.cfg` files hosted in this repo, there is minimal risk here.
+
+    Returns
+    -------
+    List[CoreParameter]
+        A list of CoreParameter objects, one for each diagnotic set.
+    """
+    config = configparser.ConfigParser()
+    config.read(cfg_path)
+    params = []
+
+    for set_name in config.sections():
+        param = SET_TO_PARAMETERS[set_name]()
+
+        for option in config.options(set_name):
+            val = config.get(set_name, option)
+            val = ast.literal_eval(val)
+
+            setattr(param, option, val)
+
+        params.append(param)
+
+    return params
```

Comment:
A new utility that converts `.cfg` diagnostic entries to parameter objects directly. This makes it easier to write integration tests using a `.cfg` file.